.github/workflows/main.yaml

```diff
@@ -0,0 +1,59 @@
+name: OpenFGA Main
```

Comment:
Can we just call this "main". The OpenFGA part goes without saying, because this lives in the openfga project.

Reply:
Sure. I'll do the same for "Pull Request". I leave the uppercase because that seems to be the convention, at least https://docs.github.com/en/actions/quickstart.